docs/Using-Fleet/Audit-Activities.md

```diff
@@ -553,14 +553,16 @@ This activity contains the following fields:
 - "host_serial": Serial number of the host.
 - "host_display_name": Display name of the host.
 - "installed_from_dep": Whether the host was enrolled via DEP.
+- "mdm_platform": Used to distinguish between Apple and Microsoft enrollments, can be "apple", "microsoft" or empty. An empty value should be considered as "apple" for backwards compatibility.
```

Comment:
Instead of "empty", let's say "" (assuming the empty value is empty string — helpful to be explicit about that). Also suggest avoiding the word "should" and using more decisive language (we're documenting how it _is_, not how it _should be_).
```suggestion
- "mdm_platform": Used to distinguish between Apple and Microsoft enrollments. Can be "apple", "microsoft" or "". If "", this value is treated as "apple" for backwards compatibility.
```

Reply:
@rachaelshaw thank you! it's not an empty string, but rather the key doesn't come at all in old fleet versions.

I pushed an update, what do you think?

Reply:
Got it! Update looks good, clear what to expect 👍